app/services/to_cocina/work/types_mapper.rb

```diff
@@ -16,7 +16,7 @@ def initialize(work_form:)
       end
 
       def call
-        return [] unless work_type
+        return [] unless work_type.presence
```

Comment:
With the change you just made to the validation callback in the work form, do we still need this check here, or are blank string work types still a thing we'll expect to see coming from other sources?

Reply:
Just being defensive.